### DISCUSSION TITLE
Jak dodać filtry do brave?

### DISCUSSION THREAD

**Discussion by 0xtrzy** [FiltersHeroes/PolishCookieConsent#63] [Pytania i odp] [answered]
jak?

**Comment by admin** [comment, marked as answer]
Jeżeli chodzi o wbudowany adblocker, to niestety nie jest to aktualnie możliwe. Musisz po prostu zainstalować uBO, AdGuard albo jakiś inny adblocker :smile: No ewentualnie kopiuj i wklej całą zawartość listy, ale to raczej nie ma sensu, bo byś musiał co jakiś czas powtarzać procedurę.
